oci/pull.bzl

```diff
@@ -129,8 +133,17 @@ def oci_pull(name, image = None, repository = None, registry = None, platforms =
                 target_name = plat_name,
                 config_path = config_path
             )
+
             if plat in _PLATFORM_TO_BAZEL_CPU:
                 platform_to_image[_PLATFORM_TO_BAZEL_CPU[plat]] = "@" + plat_name
```

Comment:
As a side note, I think a nice change here would be to:
- fail if *any* of the "platforms" specified in `oci_pull` is not present in `_PLATFORM_TO_BAZEL_CPU`
- let the user customise the `_PLATFORM_TO_BAZEL_CPU` so they can add their exotic CPUs to fix the failures above

The benefit of doing that is that we won't silently skip one of the provided platforms in the `alias` select() dict anymore.

(but that's out of scope for this PR 😄)

Reply:
wanna file an issue for it?

Reply:
Let's keep the diff here minimal so it's easy to bisect in case something goes wrong.

Reply:
Sounds good - opened #502